### ISSUE TITLE
is there a way to display message in all front ends when no execution is in progress

### ISSUE THREAD

**Issue by maddukurisrinivasarao** [Calysto/metakernel#198]
If I use metakernel to create jupyter kernel and connect to an application using REPLWrapper, is there a way to post some message in front end (may be in all front-ends) when there is no execution request from front end?

Example: 
assume, notebook connect to my_test_kernel (which is a metakernel), my_test_kernel connect to my_app (using REPLWrapper)

notebook <-> my_test_kernel <-> my_app

after some event (e.g after every 10 seconds) in my_app, how to display some message in the notebook (may be in all front-ends connected to it)

**Comment by dsblank** [comment]
I don't think so, not in the notebook. You can probably come up with a widget for JupyterLab that would allow async messages.
